DQM/Physics/src/TopSingleLeptonDQM_miniAOD.cc

```diff
@@ -373,6 +379,8 @@ void MonitorEnsemble::fill(const edm::Event& event,
   // fill monitoring plots for primary verices
   edm::Handle<edm::View<reco::Vertex> > pvs;
   if (!event.getByToken(pvs_, pvs)) return;
+  const reco::Vertex& PV= pvs->front();
```

Comment:
I will fix it. BTW, there are more modifications in same code. Is it okay sending new PR? or fix this one first?